src/lib/draw/GeometryMode.svelte

```diff
@@ -66,6 +66,13 @@
     if (mode == thisMode) {
       gjScheme.update((gj) => {
         let feature = gj.features.find((f) => f.id == currentlyEditing)!;
```

Comment:
Note that if the user loads a different file, and the arbitrary numeric feature ID happens to line up, then we'll silently replace geometry for the wrong thing. It might even change the type from a route to an area, or something silly like that. So, this fix is very much a hack, but it hopefully stops the most common error case from happening, and is better than a silent error showing up in console logs and the page being broken